src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
```

Comment:
think about utilizing try-with-resouces, it's best practice

Reply:
> think about utilizing try-with-resouces, it's best practice

but in the lectures it was said that for all crud operations except read you should not use try-with-resources since the session can be closed before using transaction.rollback(), should i use TWR anyway?